service-tests/hhvm.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'hhvm', title: 'hhvm status' });
+module.exports = t;
+
+t.create('get symfony status (default branch - not tested)')
+    .get('/symfony/symfony.json')
+    .expectJSONTypes(Joi.object().keys({ name: 'hhvm', value: 'not tested' }));
+
+t.create('get yii status (default branch - partially tested)')
+    .get('/yiisoft/yii.json')
+    .expectJSON({ name: 'hhvm', value: 'partially tested' });
+
+t.create('gets yii 1.1.19 status')
+    .get('/yiisoft/yii/1.1.19.json')
+    .expectJSON({ name: 'hhvm', value: 'partially tested' });
```

Comment:
I'm a bit concerned that these tests may be a bit brittle or tied to particular projects. I also initially found the way the service tests are done on this project a bit confusing (there's some good discussion on it [in this issue](https://github.com/badges/shields/issues/927). It can be a bit tricky when your tests depend on external services but the sweet spot we're aiming for is that we want the tests to fail if php-eye change their JSON format or move their endpoint, but the tests shouldn't tell us the service integration is broken if Yii framework decide to change their approach to testing on HHVM, for example.

I'd suggest that the valid test cases (with/without branch) should just check against a regex like `/^(tested|partially tested|not tested|maybe untested)$/`. If you also want to test the logic that leads to particular responses, you could use a test with a mocked response like this one: https://github.com/badges/shields/blob/17b4a704113e8aa8d31eee01f955206d164178c2/service-tests/homebrew.js#L19-L25 so it is less brittle.

How does that sound?